Bottom: fb245bace086a47b298557d21950e33906192964
Top:    d276687008f7391014027e262ebb77a17f028dc0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-26 04:07:07 +0100

Refresh of don-t-make-gui-drawing-calls-t

---

diff --git a/src/draw.c b/src/draw.c
index 18947dc..cef971d 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -2245,7 +2245,6 @@ ErasePolygon (PolygonTypePtr Polygon)
 void
 EraseElement (ElementTypePtr Element)
 {
-  /* set color and draw lines, arcs, text and pins */
   ELEMENTLINE_LOOP (Element);
   {
     DrawLineLowLevel (line);
